src/opencensus/proto/metrics/v1/metrics.proto

```diff
@@ -234,3 +240,42 @@ message DistributionValue {
     map<string, string> attachments = 3;
   }
 }
+
+// The start_timestamp only applies to the count and sum in the SummaryValue.
+message SummaryValue {
+  // The total number of recorded values from the beginning. Optional since
+  // some systems don't expose this.
+  google.protobuf.UInt64Value count = 1;
+
+  // The total sum of recorded values from the beginning. Optional since some
+  // systems don't expose this. If count is zero then this field must be zero
+  // or not set (if not supported).
+  google.protobuf.DoubleValue sum = 2;
+
+  // The values in this message can be reset at arbitrary unknown times, with
+  // the requirement that all of them are reset at the same time.
+  message Snapshot {
+    // The number of values in the snapshot. Optional since some systems don't
+    // expose this.
+    google.protobuf.UInt64Value count = 1;
+
+    // The sum of values in the snapshot.  Optional since some systems don't
+    // expose this. If count is zero then this field must be zero or not set
```

Comment:
How about "This field must be unset if the sum is not available."

Reply:
done.